README.md

```diff
@@ -69,17 +70,45 @@ The ConfigLoader is responsible for providing the newest port configurations and
 | [MQTT](./examples/src/sequence_to_mqtt/)                         | Routes CloudEvents that are generated from an input port to an output port, the output port publishes the events on an MQTT topic. A second router subscribes to the same topic with an MQTT port and routes them to a port which prints the event to stdout. |
 | [UNIX Socket and MQTT for armv7](./examples/src/unix_socket_and_mqtt_on_armv7/) | Routes CloudEvents that are received on an input UNIX Socket port to an output UNIX Socket port and an MQTT output port. |
 
-## Develop
+### Development Setup
 
-Different operating system setups could be fined [here](https://github.com/ce-rust/cerk/tree/master/setup).
+Different operating system setups based on Docker can be found [here](https://github.com/ce-rust/cerk/tree/master/setup).
+
+### Prerequisites
+
+1. latest version of [rustup](https://www.rust-lang.org/tools/install)
+2. Rust version 1.38.0: `rustup install 1.38.0`
+
+Optional Tooling:
+1. rustfmt: `rustup component add rustfmt`
+2. cargo-readme: `cargo install cargo-readme`
+
+### Run Tests
+
+`cargo test --all`
```

Comment:
will not work with the setup described here...

the mqtt socket needs much more...

see https://github.com/ce-rust/cerk/tree/master/setup